src/osd/PG.cc

```diff
@@ -1325,26 +1325,6 @@ bool PG::choose_acting(pg_shard_t &auth_log_shard_id, bool *history_les_bound)
     return false;
   }
 
```

Comment:
@jack-changtao Can you add an assert(!auth_log_shard->second.is_incomplete()); ?  Otherwise, I think this is right.